docs/eb_list_toolchains.rst

```diff
@@ -12,3 +12,4 @@ The list of toolchains can easily be obtained with::
 
 .. include:: version-specific/eb_list_toolchains.txt
 
+There is one special case: the `dummy` toolchain. This will not load anything and so you will be using whatever compiler can be found in the current path. If you also specify `dummy` as the toolchainversion, no dependencies will be loaded. If you leav the toolchainversion empty, the dependencies will get loaded.
```

Comment:
line way too long
also typo in `leav` (and might look a bit odd, but add examples people can copy/paste?)